docs/content/configuration/creating_volumereplication.md

```diff
@@ -12,7 +12,7 @@ When replicating a volume, be sure to follow all of the replication configuratio
 
     For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](creating_pvc.md).
 
-    **Note:** Use the `spec.csi.volumeHandle` of the relevant target PersistentVolume (PV) for the `replicationHandle` value.
+    **Important:** If you are not using policy-based replication, use the `spec.csi.volumeHandle` of the relevant target PersistentVolume (PV) for the `replicationHandle` value. If policy-based replication is being used, **do not** use the `replicationHandle` value. Using the `replicationHandle` while policy-based replication is defined in the VolumeReplicationClass produces an error message. For more information, see TROUBLESHOOTING.
```

Comment:
@ELENAGER, what about the new changes for Volume Replication with Volume Group?

Reply:
@annaparienti  volume group is being tracked separately in PR 627

Reply:
@ELENAGER  @annaparienti I need the error messages for the following so I can complete this feature:
1. In volumereplication, the replication_handle will not work with EAR.
if replication_handle is in the volumereplication handle along with replication_policy in the volumereplicationclass yaml, the log will show an error <ERROR MESSAGE - TROUBLESHOOTING>
2. system_id and replication_policy cannot be together in the same yaml. if it is, it will throw an error <ERROR_MESSAGE>
i want to add these to the troubleshooting section and link to there fromhere.

Reply:
1. got an invalid parameter: replicationHandle
2. got an invalid parameter: system_id

Reply:
@rivkap the error messages are fine

Reply:
Thanks, @ELENAGER .
@annaparienti  can we close this comment thread?